README.md

```diff
@@ -1,3 +1,65 @@
 # Python version
 
 I am using python version 3.11 and above
+
```

Comment:
Well, I don't see `.python-version` file here, but I remember we've added it. Where is it? If you add it you can remove this section